src/FormElementManagerFactory.php

```diff
@@ -11,31 +11,22 @@
 use Interop\Container\ContainerInterface;
 use Laminas\ServiceManager\AbstractPluginManager;
 use Laminas\ServiceManager\Config;
-use Laminas\ServiceManager\FactoryInterface;
+use Laminas\ServiceManager\Factory\FactoryInterface;
```

Comment:
Changing the interface is a BC break aswell.
Unless we are bumping major version here, we should avoid such changes.